src/double.js

```diff
@@ -0,0 +1,57 @@
+const ms = require('ms');
+
+export default class DoubleDecker {
+
+  constructor (one, two) {
+    this.one = one;
+    this.two = two;
+  }
+
+  get (k, cb) {
+    return this.one.get(k, (err, res) => {
+      if (err) return cb(err);
+      if (!res) return cb(null, null);
+      if (res < Date.now()) {
```

Comment:
No we are waiting for the response, but just because the cacheman engines use callbacks (cacheman itself converts all of those into a cb AND promise-based API, but the engines need to adhere to the CB interface...